Form/Form/ContentTypeType.php

```diff
@@ -121,6 +121,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                         'Descending' => 'desc',
                     ],
             ]);
+            $builder->add('translationField', ContentTypeFieldPickerType::class, [
+                'required' => false,
+                'firstLevelOnly' => true,
```

Comment:
Should check if this should be false?

Reply:
No it must be set to true, as we assume that the translation field can't be in a nested field